### ISSUE TITLE
[Firefox] Small change for radiogroup (cannot forward with Arrow Key to 2nd shipping method using keyboard)

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#7151]
**Environment Details**
Spartacus: develop branch
Browser: Chrome & Safari
Device: mac
Happens on: http://dev-com-10.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD
Calydon: **disabled**

**Steps to Reproduce**
- Add item to cart.
- Proceed to checkout
- On shipping method checkout page, try to tab to the 2nd shipping method.

**Actual Result**
Cannot tab to the 2nd shipping method. Unless we shift-tab after we've passed it.

<img width="1179" alt="Screen Shot 2020-04-02 at 10 24 03 AM" src="https://user-images.githubusercontent.com/45409515/78260679-38019680-74cc-11ea-8be0-c304ba33acae.png">

It works on Firefox.

**Comment by developpeurweb** [comment]
This is OK and is the expected behaviour, we're not supposed to tab through radio buttons individually but only **Radio Groups** and then change focus with arrow keys.

The accessibility design pattern for Radio Groups is the following:

- `TAB`: Moves focus into and out of the **radio group**.
- `Spacebar`: checks the focused radio button if it is not already checked.
- `Arrow Keys`:  moves focus to the next/previous **radio button in the group**, uncheck the previously/next focused button, and check the newly focused button.

Therefore, it does work on Chrome and Safari, however **it doesn't work for Firefox**, mainly due to how the radio buttons are wrapped in the html.

![Screen Shot 2020-04-06 at 2 45 12 PM](https://user-images.githubusercontent.com/15113219/78594451-54783880-7816-11ea-8d3d-d81007c4b15a.png)


Since this will NOT work consistently with **Screen Readers**, my suggestion is to **improve** this area following WAI-ARIA Practices e.g. `role="radiogroup"` and `role="radio" aria-checked="false"` or by refactoring and grouping `<input type="radio">` within `<fieldset>`  for the Next Accessibility milestone. https://www.w3.org/TR/wai-aria-practices/#radiobutton

**Comment by developpeurweb** [comment]
Improve **Shipping Method** area following WAI-ARIA Practices e.g. `role="radiogroup"` and `role="radio" aria-checked="false"` or by refactoring and grouping native HTML `<input type="radio">` within `<fieldset>`
Please try and implement whichever solution works best and has minimal undesired repercussions in the code.
